common/dynamicconfig/constants.go

```diff
@@ -741,6 +741,9 @@ const (
 	MaximumSignalsPerExecution = "history.maximumSignalsPerExecution"
 	// ShardUpdateMinInterval is the minimal time interval which the shard info can be updated
 	ShardUpdateMinInterval = "history.shardUpdateMinInterval"
+	// ShardUpdateMinTasksCompleted is the minimum number of tasks which must be completed before the shard info can be updated before
```

Comment:
nit: probably note that this count is across all the queues. Initially I thought it's per queue.

Reply:
Fixed